Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,49 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the `users` group (which has a `GID` of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u` _UID_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g` _GID_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s` _SHELLBIN_: The shell binary for this login. The default is `/bin/sh`. 
```

Comment:
These options should list their long names too, and value names should be the same ones as in the code.